deploy/crd/clustercompliancereports.crd.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clustercompliancereports.aquasecurity.github.io
+  labels:
+    app.kubernetes.io/managed-by: starboard
+    app.kubernetes.io/version: "0.14.1"
+spec:
+  group: aquasecurity.github.io
+  scope: Cluster
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      additionalPrinterColumns:
+        - jsonPath: .spec.specKind
+          type: string
+          name: report
+          description: The name of the complience report
+        - jsonPath: .metadata.creationTimestamp
+          type: date
+          name: Age
+          description: The age of the report
+        - jsonPath: .status.summary.failCount
+          type: integer
+          name: Fail
+          priority: 1
+          description: The number of checks that failed with Danger status
+        - jsonPath: .status.summary.passCount
+          type: integer
+          name: Pass
+          priority: 1
+          description: The number of checks that passed
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - apiVersion
+            - kind
+            - metadata
+            - spec
+          properties:
+            apiVersion:
+              type: string
+            kind:
+              type: string
+            metadata:
+              type: object
+            spec:
+              type: object
+              required:
+                - specKind
+                - name
+                - description
+                - version
+                - cron
+                - controls
+              properties:
+                specKind:
+                  type: string #type of report example compliance
+                name:
+                  type: string
+                description:
+                  type: string
+                version:
+                  type: string
+                cron:
+                  type: string
+                  pattern: '(@(annually|yearly|monthly|weekly|daily|hourly|reboot))|(@every (\d+(ns|us|µs|ms|s|m|h))+)|((((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*) ?){5,7})'
+                controls:
+                  type: array
+                  items:
+                    type: object
+                    required:
+                      - name
+                      - id
+                      - resources
+                      - mapping
+                    properties:
+                      name:
+                        type: string
+                      description:
+                        type: string
+                      id: # control check id
```

Comment:
Instead of using comments we can explain the meaning of each property by describing it. See how it's done for vulnerabilityreports.
```suggestion
                      id:
                        description: "control check id"
```

This would allows us to explain the CRD with kubectl, e.g. describe the `cron` property:

```console
$ kubectl explain clustercompliancereports.spec.cron
KIND:     ClusterComplianceReport
VERSION:  aquasecurity.github.io/v1alpha1

FIELD:    cron <string>

DESCRIPTION:
     <empty>
```